app/gui/view/debug_scene/interface/src/lib.rs

```diff
@@ -232,7 +234,7 @@ fn init(app: &Application) {
         }
     });
 
-    project_view.show_prompt();
+    root_view.project().show_prompt();
```

Comment:
Might be broken, check develop for regressions

Reply:
Develop (before any Welcome Screen changes) looks exactly the same way, so no regressions were introduced by Welcome Screen.